.github/workflows/release-part-1.yml

```diff
@@ -0,0 +1,123 @@
+name: Release (Part 1 of 2)
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        required: true
+        type: string
+
+env:
+  VERSION: ${{ inputs.version }}
+  AUTORELEASE_BRANCH: autorelease/${{ inputs.version }}
+  # use a personal access token here which has permissions to trigger further actions
+  # this is necessary for the pr checks
+  GITHUB_TOKEN: ${{ secrets.AUTORELEASE_BOT_PAT }}
+
+# permissions:
+#   contents: write
+#   pull-requests: write
+#   actions: write
```

Comment:
Do we want to remove these, or are they useful to keep around?

Reply:
Oops! Fixed